lib/optimizely/config/datafile_project_config.rb

```diff
@@ -281,6 +302,52 @@ def get_variation_from_id(experiment_key, variation_id)
       nil
     end
 
+    def get_variation_from_id_by_experiment_id(experiment_id, variation_id)
```

Comment:
Can variation ids duplicate across experiments? why do we need `experiment_id` if we have `variation_id`. shouldn't `variation_id` be enought to uniquely identify and fetch a variation?

Reply:
Yes as far as I know variation Ids are unique but I was following this method [`def get_variation_from_id(experiment_key, variation_id)`](https://github.com/optimizely/ruby-sdk/blob/697e13d9a67c2f4ddde3764898bbd68b1d9b52af/lib/optimizely/config/datafile_project_config.rb#L282)